src/docfx/docset/Document.cs

```diff
@@ -6,11 +6,14 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Text;
 
 namespace Microsoft.Docs.Build
 {
     internal class Document
     {
+        private readonly string _content;
```

Comment:
Don't change `Document`. Just change `Resolve.TryResolveContent` it will cover both token and code snippet.

Reply:
The document has ReadContent and ReadText and we can not make sure the they are only used in TryResolveContent

---
In reply to: [236350421](https://github.com/dotnet/docfx/pull/3677#discussion_r236350421) [](ancestors = 236350421)